test/integration_test.go

```diff
@@ -280,6 +281,40 @@ func TestTerragruntProviderCache(t *testing.T) {
 
 }
 
+func TestTerragruntDestroyOrder(t *testing.T) {
+	t.Parallel()
+
+	cleanupTerraformFolder(t, TEST_FIXTURE_DESTROY_ORDER)
+	tmpEnvPath := copyEnvironment(t, TEST_FIXTURE_DESTROY_ORDER)
+	rootPath := util.JoinPath(tmpEnvPath, TEST_FIXTURE_DESTROY_ORDER, "app")
+
+	runTerragrunt(t, fmt.Sprintf("terragrunt run-all apply --terragrunt-non-interactive --terragrunt-working-dir %s", rootPath))
+
+	showStdout := bytes.Buffer{}
+	showStderr := bytes.Buffer{}
+
+	err := runTerragruntCommand(t, fmt.Sprintf("terragrunt run-all destroy --terragrunt-non-interactive --terragrunt-working-dir %s", rootPath), &showStdout, &showStderr)
```

Comment:
I think it is better to use `runTerragruntCommandWithOutput()` it will return the output results and will also log outputs to terminal which will help to debug the issue

Reply:
Redone. Thanks.